### DISCUSSION TITLE
Support - EJBCA 7.11.0 Community

### DISCUSSION THREAD

**Discussion by kfox13** [Keyfactor/ejbca-ce#650] [General]
Hi,
We're currently not using the latest version of EJBCA as we currenttly have users and a few machines that are using functionality within the public web interface and so to move over to RA web would take some training / dev work.

We have just done a threat model on our current setup with our Infosec team and are wondering what would happen if there was say a vulnerability found in public web. 
If this were to occur would there be patches available so that we could continue running securely? or would our only path to be to upgrade to the latest version of EJBCA (where public web no longer exists)?

Thanks

**Comment by primetomas** [comment]
Hi,
There would be no official update to EJBCA Community 7.11 from the project, only to current versions. You would have to patch that yourself.

Cheers,
Tomas
